tasks/rvm.yml

```diff
@@ -36,6 +36,15 @@
   shell: curl -sSL https://rvm.io/mpapis.asc | gpg --import -
   when: not ansible_check_mode and rvm1_gpg_keys != '' and gpg_result.rc != 0
 
+- name: Install which (CentOS, RHEL)
+  become_user: root
```

Comment:
you know what this task will break environments where rvm is installed for non-privileged users.

Reply:
You are right, I should have been more explicit about this aspect of the change in my description. Given the existence of assumptions about the installed packages in [rvm-installer](https://github.com/rvm/rvm/blob/master/binscripts/rvm-installer#L33), it really boils down to what the contract of this role should be. 

1. either the role should remain fully functional for non-privileged users and it is the responsibility of the consumer to install the prerequisites on their distribution, i.e. maintain the current statu quo 
2. or the role should unconditionally install its prerequisites using a privileged user. This is  the implementation of the last commit and effectively the most breaking change as you highlighted in this comment
3. a third option would be to allow errors in the installation task via `ignore_errors: true`. While this should maintain compatibility for existing non-privileged user, it would then delegate the responsibility to the consumers to grant the privileges to the `rvm1_user` appropriately to enable the functionality, either by using `root` or by adding `rvm1_user` to `sudoers` as mentioned in https://github.com/rvm/rvm1-ansible/pull/165#issuecomment-366734108.

Reply:
@sbesson We can try the option `ignore_errors` and add 
```
when: {{ rvm1_user }} == root and ansible_distribution == 'CentOS' or ansible_distribution == 'Red Hat Enterprise Linux'
```
but we may need to move this task to another file called for example  `system-dependencies.yml`.